avocado/core/job.py

```diff
@@ -319,10 +319,14 @@ def _log_avocado_version():
         job_log = _TEST_LOGGER
         job_log.info('Avocado version: %s', version.VERSION)
         if os.path.exists('.git') and os.path.exists('avocado.spec'):
-            cmd = "git show --summary --pretty='%H' | head -1"
-            status, top_commit = commands.getstatusoutput(cmd)
+            cmd = "git show --summary --pretty='%H'"
+            result = process.run(cmd, ignore_status=True)
```

Comment:
The utils.process is very useful and powerful tool. On the other hand it has a great overhead, so we should always balance and use the right tool. For simple standalone execution I'd prefer simple solution...